code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -80,6 +80,28 @@
 		return
 	A.emag_act(user)
 
+/obj/item/card/cmag
+	desc = "It's a card coated in a slurry of electromagnetic bananium."
+	name = "jestographic sequencer"
+	icon_state = "cmag"
+	item_state = "card-id"
+	origin_tech = "magnets=2;syndicate=2"
+	flags = NOBLUDGEON
+	flags_2 = NO_MAT_REDEMPTION_2
+
+/obj/item/card/cmag/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/slippery, src, 16 SECONDS, 100)
+
+/obj/item/card/cmag/attack()
+	return
+
+/obj/item/card/cmag/afterattack(atom/target, mob/user, proximity)
+	var/atom/A = target
+	if(!proximity)
+		return
+	A.cmag_act(user)
```

Comment:
```suggestion
	if(!proximity)
		return
	target.cmag_act(user)
```
no need to declare another var here, it's redundant, (unless this was done for a specific reason?)